addons/mass_mailing/wizard/mailing_mailing_test.py

```diff
@@ -11,10 +11,14 @@ class TestMassMailing(models.TransientModel):
     _name = 'mailing.mailing.test'
     _description = 'Sample Mail Wizard'
 
-    email_to = fields.Text(string='Recipients', required=True,
-                           help='Carriage-return-separated list of email addresses.', default=lambda self: self.env.user.email_formatted)
+    email_to = fields.Text(string='Recipients', required=True, default=lambda self: self._get_email_to(),
```

Comment:
This field could then become a related field (readonly=False) based on `mass_mailing_id.default_test_email_to`.
And so we can also remove the `_get_email_to` below